meta-resin-common/recipes-core/dropbear/dropbear_%.bbappend

```diff
@@ -29,12 +29,10 @@ do_install_append() {
         echo 'DROPBEAR_EXTRA_ARGS="-s"' >> ${D}/etc/default/dropbear
     fi
 
```

Comment:
Maybe the git log should detail why we have this change. Having just this git log entry, looking back in the future it will not be clear why this change is done